### PR TITLE
Serialized bag path as link

### PR DESCRIPTION
**Pull request by nikathone**
# What does this Pull Request do?

Display serialized bag path as link to allow user to click the path and trigger the download of the bag.

### CODE REVIEW

**Comment by nikathone** [comment]
Also was thinking if we shouldn't just link `Serialized collection Bag created and saved` instead of displaying the entire path.

**Comment by mjordan** [comment]
Hi @nikathone, thank you for pointing out the lack of a download link for collection bags. 

We normally open a ticket in our [JIRA](https://jira.duraspace.org) before opening a PR. Also, all contributors to the Islandora codebase must sign a [Contributor License Agreement](https://islandora.ca/resources/contributors) with the Islandora Foundation. I'm travelling at the moment and can't check easily whether you have signed one.

Code freeze for the upcoming release is this Wednesday (March 30) so I'd like to propose that if you haven't signed a CLA, I can open a JIRA ticket for this, implement your change and acknowledge you in my commit message. Does that sound OK?

**Comment by nikathone** [comment]
@mjordan FYI I did fill and submit the SLA. Also do you know why travis is failing?

**Comment by mjordan** [comment]
@nikathone I'm sorry I didn't check for a CLA. I see yours now.

As for travis, it's showing `The command "drush coder-review --reviews=production,security,style,i18n,potx,sniffer islandora_bagit" exited with 1.` I'll take a closer look at this and provide code review over the weekend.

**Review by mjordan — COMMENTED** [review]
[please ignore, replaced with clearer version, below]

**Review by mjordan — COMMENTED** [review]
It's best to use Drupal's `l()` function to generate links, e.g. `      drupal_set_message(t('Serialized collection Bag created and saved at !link', array(
        '!link' => l($serialized_bag_path, $serialized_bag_path),`

**Comment by mjordan** [comment]
@nikathone only suggestion I have on the code is to use Drupal's `l()` function instead of hard-coding HTML. I provided some example code.

Also, we like to have JIRA issues before pull requests. I've opened one for this PR at https://jira.duraspace.org/browse/ISLANDORA-2194. If you open a PR, please also use the PR template provided in every Islandora repo.

**Comment by nikathone** [comment]
@mjordan using `l()` that what I was doing in the previous commit and was causing travis to fail. https://www.drupal.org/node/322774 advice to use hard coded html.
TL;DR for the link above
```
// Bad examples:
// DO NOT DO THESE THINGS
$BAD_EXTERNAL_LINK = t('Look at Drupal documentation at !handbook.', array(
  '!handbook' => '<a href="http://drupal.org/handbooks">'. t('the Drupal Handbooks') .'</a>',
));
$ANOTHER_BAD_EXTERNAL_LINK = t('Look at Drupal documentation at <a href="http://drupal.org/handbooks">the Drupal Handbooks</a>.');
$BAD_INTERNAL_LINK = t('To get an overview of your administration options, go to !administer in the main menu.', array(
  '!administer' => l(t('the Administer screen'), 'admin'),
);

// Good examples:
// Do this instead.
$external_link = t('Look at Drupal documentation at <a href="@drupal-handbook">the Drupal Handbooks</a>.', array(
  '@drupal-handbook' => 'http://drupal.org/handbooks',
));
$internal_link = t('To get an overview of your administration options, go to <a href="@administer-page">the Administer screen</a> in the main menu.', array(
  '@administer-page' => url('admin'),
));

**Comment by nikathone** [comment]
Also went to https://jira.duraspace.org/browse/ISLANDORA-2194 and it seems to be for `Option to display Compound parent metadata instead of first child`. I guess I will also need to request an account creation in order to be able to create an issue right?

**Comment by mjordan** [comment]
@nikathone Sorry about the JIRA link, that was my mistake. The correct one is https://jira.duraspace.org/browse/ISLANDORA-2195.

I had no idea that using `l()` made travis angry. I'd like to ask @Islandora/7-x-1-x-committers if advice on that should be in our "styleguide" (since we are currently reviewing that).

Yes, our standard practice for opening PRs is outlined in the "Contributing code" section of https://github.com/Islandora/islandora/blob/7.x/CONTRIBUTING.md, so you'll need to get a JIRA account.

**Comment by rosiel** [comment]
Good catch @nikathone - I wasn't sure why it was upset about the links, but that documentation you found makes sense. I'll toss it in our style guide, good suggestion @mjordan.

**Comment by mjordan** [comment]
OK, I will merge tomorrow, allowing 24 hours for additional feedback.

**Review by mjordan — APPROVED** [review]

**Comment by mjordan** [comment]
Merged. Thanks @nikathone for your patience and for adding this fix.

**Comment by DiegoPino** [comment]
@mjordan I read 

>  for adding this fix.

fix == Bug? :smile: or is this an improvement as JIRA ticket says? Not asking for changes, just want to make sure we don't lose a release pull, thanks!

**Comment by mjordan** [comment]
@DiegoPino improvement. Sorry if I fumbled that.

**Comment by DiegoPino** [comment]
No worries, was just checking!
